packages/sdk/src/transactions/psbt.ts

```diff
@@ -1,10 +1,10 @@
-import * as ecc from "@bitcoinerlab/secp256k1";
-import BIP32Factory, { BIP32API } from "bip32";
-import { Network, Psbt } from "bitcoinjs-lib";
+import * as ecc from "@bitcoinerlab/secp256k1"
+import BIP32Factory, { BIP32API } from "bip32"
+import { Network, Psbt } from "bitcoinjs-lib"
 
-import { OrditApi } from "../api";
-import { createTransaction, getNetwork } from "../utils";
-import { GetWalletOptions, getWalletWithBalances } from "../wallet";
+import { OrditApi } from "../api"
+import { createTransaction, getNetwork } from "../utils"
+import { GetWalletOptions, getWalletWithBalances } from "../wallet"
 
 export async function createPsbt({
```

Comment:
Can we have `finalize` as a parameter here as well for the non-wallet `createPsbt` function?

Reply:
Hmm, sure.

Reply:
#31

Reply:
@jordankzf Added `finalize` to `ordit.unisat.signPsbt` method; https://github.com/sadoprotocol/ordit-sdk/pull/30/commits/7e7d9ca86447f15880678ad805f8202b20954732

Adding the same to `createPsbt` is not possible since a PSBT can be finalized only after all its inputs are signed.